index.js

```diff
@@ -104,6 +104,7 @@
 
   //  identity :: a -> a
   function identity(x) { return x; }
+  Object.freeze(identity);
```

Comment:
There's no need to freeze this object I think. Plus it may be increasing our minimum JS run-time requirements, but I'm not sure about that.